test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
+
+ENV password=notpassword1
+ENV dbpass=notpassword2
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- d7a3f186-2c67-4991-83d0-35d757d8aa1b -->